src/components/flash/flash.js

```diff
@@ -223,19 +233,23 @@ class Flash extends React.Component {
    * @param {Object} ev
    * @return(Void)
    */
-  toggleDialog = (key, ev) => {
-    if (ev) { ev.preventDefault(); }
-
-    let state = this.state.dialogs[key];
-    // open/close the dialog
-    this.setState({ dialogs: { [key]: !state }});
-
-    // start/stop the timer if the dialog opens or closes
-    if (state) {
-      this.startTimeout();
-    } else {
-      this.stopTimeout();
-    }
+  toggleDialog = (key) => {
```

Comment:
Yeah sure, using the fat arrow means we can avoid defining the bind in the constructor because it automatically binds `this`. Less boilerplate code.

I have seen a few places recommend defining the bind in the constructor, but this is mostly in code where they are not adopting `stage-0` features (but you are in Carbon).